File: 30 Days of Code/Day 26/NestedLogic.py
returned=list(map(int,input().split()))
rday,rmonth,ryear=returned

due=list(map(int,input().split()))
dday,dmonth,dyear=due
fine=0

# Start of Function
if ryear>dyear:
    fine=10000
elif ryear==dyear:
    if rmonth>dmonth:
        fine=500*(dmonth-rmonth)
    elif dmonth==rmonth and rday>dday:
        fine=15*(rday-dday)

print(fine)

File: 30 Days of Code/Day 26/NestedLogic.cpp
#include <cmath>
#include <cstdio>
#include <vector>
#include <iostream>
#include <algorithm>
using namespace std;


int main() {
    /* Enter your code here. Read input from STDIN. Print output to STDOUT */   
    int dday, rday, dmonth, rmonth, dyear, ryear;
    int fine=0;
    cin>>rday>>rmonth>>ryear;
    cin>>dday>>dmonth>>dyear;

    if (ryear>dyear) fine=10000;
    else if (ryear==dyear){
        if (rmonth>dmonth) fine=500*(rmonth-dmonth);
        else if (rmonth==dmonth && rday>dday) fine=15*(rday-dday);
    }
    cout<<fine;
    return 0;
}
